src/transformers/convert_slow_tokenizer.py

```diff
@@ -1283,6 +1284,9 @@ def post_processor(self):
 
 class GemmaConvert(SpmConverter):
     handle_byte_fallback = True
+    SpmExtractor = GemmaSentencePieceExtractor
+    # start and end of turn tokens must be marked as special
+    special_tokens = {"<start_of_turn>", "<end_of_turn>"}
```

Comment:
ah but only for the chat version no?

Reply:
Is there a way to only apply it for the chat version? 👀 If not, then I think this is okay because these tokens - by their definition - are only used in the chat version, so users of the base model won't be affected unless they are finetuning (in which case, these tokens should be marked as special).